Exercises/JS/07-js-loops/js/script.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+for (let i = 1; i < 1000; i++) {
+   let result = +i;
```

Comment:
The operators should be swapped here. When you do this, the value of `i` will be assigned to result.
Objective of this exercise is to break the loop when the intermediate result exceeds 400.
If you want the value of `i` to be added to `result`, then the variable should be declared elsewhere.
Keep in mind,
`result += i` is the shortened form of `result = result + i` and for this, result should be declared somewhere else. Hope I'm clear if not please let me know.

Reply:
I understood and fixed it.